app/security/Roles.scala

```diff
@@ -182,7 +182,7 @@ object Roles {
 
   val userJourneySequence: List[(CsrAuthorization, Call)] = List(
     ApplicationStartRole -> routes.HomeController.present,
-    PersonalDetailsRole -> routes.FastStreamApplication.generalDetails(None),
+    PersonalDetailsRole -> routes.PersonalDetailsController.present(None),
```

Comment:
Roles are there to stop a user navigating to a page too far along their journey. E.g. you can't chose schemes until you have filled in your personal details. Also, you can't submit until you have completed diversity questions